scripts/e2e-test.sh

```diff
@@ -78,3 +78,28 @@ helm template deploy/helm/eunomia-operator/ \
   --set eunomia.operator.image.tag=dev \
   --set eunomia.operator.namespace=$TEST_NAMESPACE | kubectl delete -f -
 
+# Below block is to ensure hello-world-yaml example is working
+helm template deploy/helm/eunomia-operator/ \
+  --set eunomia.operator.image.tag=dev \
+  --set eunomia.operator.image.pullPolicy=Never | kubectl apply -f -
+
+kubectl create namespace eunomia-hello-world-yaml-demo
+
+kubectl apply -f examples/hello-world-yaml/eunomia-runner-sa.yaml -n eunomia-hello-world-yaml-demo
+
+kubectl apply -f examples/hello-world-yaml/cr/hello-world-cr1.yaml -n eunomia-hello-world-yaml-demo
+
+timeout=30
+
+while ((--timeout)) && [[ "$(kubectl get po -n eunomia-hello-world-yaml-demo -l name=hello-world -o=jsonpath="{range .items[*]}{.status.phase}{'\n'}{end}")" != "Running"  ]];do
+  echo "waiting for hello-world deployment: remaining $timeout sec..."
+  sleep 1
+done
+if [[ $timeout == 0 ]]; then
+  echo "Example CR1 Test FAILED"
```

Comment:
please write full name of the example also in this message ("hello-world-yaml-demo") - we plan to test other examples in future too, so this will be helpful to know which one of them is failed